presto-common/src/main/java/com/facebook/presto/common/function/SqlFunctionProperties.java

```diff
@@ -139,8 +153,12 @@ public static Builder builder()
         private long sessionStartTime;
         private Locale sessionLocale;
         private String sessionUser;
+        private Map<String, String> extraCredentials;
 
-        private Builder() {}
+        private Builder()
+        {
+            this.extraCredentials = new HashMap<>();
+        }
```

Comment:
inline the creation of `extraCredentials` with `EMPTY_MAP` from java base collections